app/libs/backbone-forms/distribution.amd/backbone-forms.js

```diff
@@ -825,7 +825,8 @@ Form.Field = Backbone.View.extend({
       fieldAttrs: schema.fieldAttrs,
       editorAttrs: schema.editorAttrs,
       key: this.key,
-      editorId: this.editor.id
+      editorId: this.editor.id,
+      self: this
```

Comment:
Are you sure it's a good idea to modify the library code? You couldn't do anything else?

Reply:
it was a bad move from the begining to version the libraries but we can't revert without breaking stuff so I guess it's ok for now. We can think of a better solution in the future.

Reply:
I resolved to this as a last resort after exploring all possible solutions. Even tried to upgrade backbone-forms. But after the upgrade the whole forms were broken. The code seems to do already a lot of custom stuff.
It's relatively safe in the sense that backbone-forms is not maintained any more, so we don't risk being completely out-of-date.